app/src/main/res/layout-v21/layout_search.xml

```diff
@@ -103,21 +103,45 @@
 
         </HorizontalScrollView>
 
-        <TextView
-            android:id="@+id/searchDeepSearchTV"
-            android:layout_width="wrap_content"
+        <LinearLayout
+            android:id="@+id/deepSearchContainer"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:background="?attr/selectableItemBackground"
-            android:clickable="true"
-            android:focusable="true"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginEnd="8dp"
+            android:layout_marginBottom="8dp"
+            android:gravity="center_vertical"
+            android:orientation="horizontal"
             android:padding="8dp"
-            android:textSize="16sp"
             android:visibility="gone"
-            app:layout_constraintBottom_toTopOf="@id/searchResultsHintTV"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintHorizontal_bias="0"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/searchRecentItemsScrollView" />
+            app:layout_constraintTop_toBottomOf="@id/searchRecentItemsScrollView">
+
+            <TextView
+                android:id="@+id/notFindingTextView"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1.3"
+                android:text="@string/not_finding_what_you_re_looking_for"
+                android:textSize="16sp" />
+
+            <androidx.appcompat.widget.AppCompatButton
```

Comment:
pls update the button such that there is some external margin and reduce the internal padding a bit. currently, the button looks too out of place.